bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoBindingConstants.java

```diff
@@ -12,292 +12,124 @@
-                    EventTypeEnum.TAG_SMALL_MOVE, EventTypeEnum.TAG_UNINSTALLED, EventTypeEnum.TAG_OPEN)
-            .collect(Collectors.toSet());
-    public static final Set<EventTypeEnum> PERSON_EVENTS = Stream.of(EventTypeEnum.PERSON, EventTypeEnum.PERSON_AWAY)
-            .collect(Collectors.toSet());
-    public static final Set<EventTypeEnum> PRESENCE_EVENTS = Stream
-            .of(EventTypeEnum.OUTDOOR, EventTypeEnum.ALIM, EventTypeEnum.DAILY_SUMMARY).collect(Collectors.toSet());
+    // Things properties
+    public static final String PROPERTY_CITY = "city";
+    public static final String PROPERTY_COUNTRY = "country";
+    public static final String PROPERTY_TIMEZONE = "timezone";
+    public static final String PROPERTY_FEATURE = "feature";
+
+    // Channel group ids
+    public static final String GROUP_LAST_EVENT = "last-event";
+    public static final String GROUP_TEMPERATURE = "temperature";
+    public static final String GROUP_HUMIDITY = "humidity";
+    public static final String GROUP_AIR_QUALITY = "airquality";
+    public static final String GROUP_NOISE = "noise";
+    public static final String GROUP_PRESSURE = "pressure";
+    public static final String GROUP_TIMESTAMP = "timestamp";
+    public static final String GROUP_RAIN = "rain";
+    public static final String GROUP_WIND = "wind";
+    public static final String GROUP_ENERGY = "energy";
+    public static final String GROUP_SIGNAL = "signal";
+    public static final String GROUP_BATTERY = "battery";
+    public static final String GROUP_SECURITY = "security";
+    public static final String GROUP_CAM_STATUS = "status";
+    public static final String GROUP_CAM_LIVE = "live";
+    public static final String GROUP_PRESENCE = "presence";
+    public static final String GROUP_PERSON = "person";
+    public static final String GROUP_PERSON_EVENT = "person-event";
+    public static final String GROUP_ROOM_TEMPERATURE = "room-temperature";
+    public static final String GROUP_ROOM_PROPERTIES = "room-properties";
+    public static final String GROUP_TH_PROPERTIES = "th-properties";
+    public static final String GROUP_TH_SETPOINT = "setpoint";
+    public static final String GROUP_LOCATION = "location";
+
+    // Alternative extended groups
+    public static final String GROUP_EXTENSION = "-extended";
+    public static final String GROUP_EXTENDED_TIMESTAMP = GROUP_TIMESTAMP + GROUP_EXTENSION;
+    public static final String GROUP_BATTERY_EXTENDED = GROUP_BATTERY + GROUP_EXTENSION;
+    public static final String GROUP_PRESSURE_EXTENDED = GROUP_PRESSURE + GROUP_EXTENSION;
+    public static final String GROUP_TEMPERATURE_EXTENDED = GROUP_TEMPERATURE + GROUP_EXTENSION;
+    public static final String GROUP_AIR_QUALITY_EXTENDED = GROUP_AIR_QUALITY + GROUP_EXTENSION;
+    public static final String GROUP_TEMPERATURE_OUTSIDE = GROUP_TEMPERATURE + "-outside";
```

Comment:
Is this newly introduced "-outside" suffix needed? I means that channel is now `netatmo:outdoor:home:indoor:outdoor:temperature-outside#value,` for instance. Since temperature is in context of an outdoor module, it seems redundant. If there is reason to keep it, README.md must be updated to reflect this.

Reply:
You're correct, I did not think of this impact. TBH I need to think of it twice, as I use generic groups aggregated for each module, directing to the appropriate channel type I'd is not so simple.

Reply:
Thanks for addressing this.